library/src/main/java/org/xmtp/android/library/messages/InvitationV1.kt

```diff
@@ -38,9 +39,8 @@ class InvitationV1Builder {
 fun InvitationV1.createRandom(context: Invitation.InvitationV1.Context? = null): InvitationV1 {
     val inviteContext = context ?: Invitation.InvitationV1.Context.newBuilder().build()
     val randomBytes = SecureRandom().generateSeed(32)
-    val randomString =
-        Base64.decode(randomBytes, 0).toByteString().toString().replace(Regex("=*$"), "")
-            .replace(Regex("[^A-Za-z0-9]"), "")
+    val randomString = encodeToString(randomBytes, 0).replace(Regex("=*$"), "")
+        .replace(Regex("[^A-Za-z0-9]"), "")
```

Comment:
I keep getting this backwards when reading iOS code. But this should be encoding and a test found that 😄